utils/lldb/lldbSwiftDataFormatters.py

```diff
@@ -175,3 +178,7 @@ def DemangleNodeSummaryProvider(valobj, internal_dict):
         result += str(num_children)
         result += ' children'
     return result
+
+
+def IdentifierSummaryProvider(valobj, internal_dict):
+    return valobj.GetChildMemberWithName('Pointer').GetSummary()
```

Comment:
lldb also has a [summary string](https://lldb.llvm.org/use/variable.html#summary-strings) syntax. As an alternative to using a python function, this summary could be defined as:

```
type summary add -w swift -s "${var.Pointer%S}" swift::Identifier
```

Where the `%S` is the operator equivalent of `GetSummary()`.

I am fine with either method, but figured I'd point it out.

Reply:
Thanks! Changed it to the more concise syntax.